tests/vmi_configuration_test.go

```diff
@@ -2849,11 +2849,11 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			By("Check values in dmidecode")
 			// Check on the VM, if expected values are there with dmidecode
 			Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
-				&expect.BSnd{S: fmt.Sprintf("[ $(sudo dmidecode -s system-family | tr -s ' ') = %s ] && echo 'pass'\n", smBIOS.Family)},
+				&expect.BSnd{S: fmt.Sprintf("[ \"$(sudo dmidecode -s system-family | tr -s ' ')\" = \"%s\" ] && echo 'pass'\n", smBIOS.Family)},
```

Comment:
nit: Go has a dedicated feature to avoid the escaping backslash

```suggestion
				&expect.BSnd{S: fmt.Sprintf(`[ "$(sudo dmidecode -s system-family | tr -s ' ')" = "%s" ] && echo 'pass'\n`, smBIOS.Family)},
```